### DISCUSSION TITLE
Add to AdGuard's hostlist registry

### DISCUSSION THREAD

**Discussion by balupton** [blocklistproject/Lists#985] [Ideas]
Would be great if all the filters here are added to the @AdguardTeam's hostlist registry:
https://github.com/AdguardTeam/HostlistsRegistry

Which is used in their adguard-dns product:
https://adguard-dns.io/en/dashboard/settings/blocklist

For me, I would use this to replace the builtin family filter which blocks more than just porn (for instance, some university website courses get blocked due to adult themes). So I'd want to adjust my adguard-dns settings to not use the adguard family filter, but instead use the porn blocklist from here.

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/61148/235844560-703d2d4a-48e6-4730-843e-5cc2931384cc.png">

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/61148/235844620-cd058318-bace-4ad6-a464-8010f0147907.png">

**Comment by gap579137** [comment]
I’ll see what I can do to get them included.
